catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -650,5 +657,6 @@
         <feature>catalog-metacardingest-network</feature>
         <feature>catalog-core-attachment</feature>
         <feature>catalog-plugin-gazetteer</feature>
+        <feature>data-migration</feature>
```

Comment:
Ideally each migration would also specify a version that it affects so that these migrations could be run intelligently and automatically.

Reply:
I think that would be okay as long as `workspace-query-separation` is moved to its own feature. Probably better for administrators to install features rather than individual bundles.

Reply:
Done. This feature needs to be installed through the admin panel to be used now